include/battle_ai_script_commands.h

```diff
@@ -16,6 +16,25 @@ enum
     BATTLEAI_DO_NOT_PROCESS
 };
 
+
+enum
```

Comment:
These can probably just be a series of #defines

Reply:
Sure, its changed :)

I wasn't sure which style to go for since the rest of the file used enum and both methods result in identical SHA1s